### ISSUE TITLE
Add a new record fluid content in Page mode problem

### ISSUE THREAD

**Issue by metrgonzoo** [FluidTYPO3/flux#849]
Hi everyone,

I'm using Typo3 V6.2.13, flux 7.2.1 and fluidcontent 4.2.2
I have a problem in page mode.
When I add a new record (fluid content) on a empty page, it's work fine. 
But, if a record exist on this page and I add a new record, the form is not displayed (only if I add after the first record). And the fluid content type list  is empty.

Someone have the same problem?

![flux](https://cloud.githubusercontent.com/assets/11647914/8251493/4a6be170-167d-11e5-959a-0db391296459.jpg)

**Comment by erredeco** [comment]
It happens often to me too, even on lesser versions. I think it is a cache related problem, quite annoying, you have to delete all cache from Install Tool

**Comment by morinfa** [comment]
Clearing the cache has no effect for me.
This problem occurs when there are multiple FCE with more than one group defined (in the "configuration" section, attribute "options").

**Comment by morinfa** [comment]
Clearing the cache has not effect for me.
This problem occurs for the FCE which are not located in the Resources/Private/Templates/Content/ directory.

**Comment by NamelessCoder** [comment]
Could you give a bit more detail about your setup - how is your TS defined re: template paths? Is this an empty page that is properly inside the page tree and "knows" about your TS?

Also, if you "reload frame" (**not** the entire window!) then is the "Fluid Content type" selector filled with options?

**Comment by metrgonzoo** [comment]
My TS Config is:
[tx_flux]
   [view]
    [templateRootPath] = EXT:flux/Resources/Private/Templates/
    [partialRootPath] = EXT:flux/Resources/Private/Partials/
    [layoutRootPath] = EXT:flux/Resources/Private/Layouts/

If my page has no record, it's work. 
But, if my page has already a record (any, not only fluid content) and I want to Add record after this, then the form is not displayed.

In my Ext my path to the fluid content is : myExt/Resources/Private/Templates/modules/Simple/Content and not the default path : myExt/Resources/Templates/Content

If I reload frame, the selector has no changed, but if I save the page, the selector is complet.

**Comment by metrgonzoo** [comment]
Sorry my all TS Config is : 

plugin.tx_myExt {
	view {
		templateRootPaths {
			10 = EXT:myExt/Resources/Private/Templates/
			20 = EXT:myExt/Resources/Private/Templates/Footer/
			30 = EXT:myExt/Resources/Private/Templates/Modules/Simple/
			40 = EXT:myExt/Resources/Private/Templates/Modules/Complex/
			50 = EXT:myExt/Resources/Private/Templates/Products/Watch/
			60 = EXT:myExt/Resources/Private/Templates/Products/Jewellery/
			70 = EXT:myExt/Resources/Private/Templates/Products/Accessories/
			80 = EXT:myExt/Resources/Private/Templates/Popins/
		}
		partialRootPaths {
			10 = EXT:myExt/Resources/Private/Partials/
		}
		layoutRootPaths {
			10 = EXT:myExt/Resources/Private/Layouts/
		}
		overlays {
			templateRootPath = EXT:myExt/Resources/Private/Overlays/Templates/
			partialRootPath = EXT:myExt/Resources/Private/Overlays/Partials/
			layoutRootPath = EXT:myExt/Resources/Private/Overlays/Layouts/
		}
	}
}

**Comment by NamelessCoder** [comment]
Hmm, maybe you misunderstand how to add template paths. You don't set these paths for Flux, which is what you did - you must set them for your own extension. Furthermore, you should at this point be using the plural naming of template paths. If you don't set the alternative path, in the context of your extension, Flux won't be able to detect the templates. Furthermore, your alternative path contains one segment which is using an underscore starting letter - this must be an uppercase starting letter (but may not present symptoms depending on your file system's case sensitivity).

To recap: you register an extension key in Flux and Flux then reads the templateRootPaths configured for that extension key and scans for available content elements. If either the extension key registration or path definitions are wrong or missing, your symptom would happen and I'm guessing one of these is the explanation.

**Comment by bjo3rnf** [comment]
Maybe related to https://github.com/FluidTYPO3/fluidcontent/issues/227

**Comment by xf-** [comment]
Can you test this issue against current development branch?

**Comment by NamelessCoder** [comment]
Resolved by adding the Flux-related column names to the "defaultValues" array in TCA. The values are now copied when you use the "create new after" button so they get the same values as the previous element.
